lib/tracker.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'transaction'
+
+class Tracker
```

Comment:
`Tracker` has two responsibilities.  What are they? Could they be separated into separate classes?